framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/KieServerWithExternalDatabaseScenarioBuilderImpl.java

```diff
@@ -0,0 +1,43 @@
+package org.kie.cloud.openshift.scenario.builder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.scenario.KieServerWithExternalDatabaseScenario;
+import org.kie.cloud.api.scenario.builder.KieServerWithExternalDatabaseScenarioBuilder;
+import org.kie.cloud.api.scenario.builder.WorkbenchWithKieServerScenarioBuilder;
+import org.kie.cloud.openshift.OpenShiftController;
+import org.kie.cloud.openshift.constants.OpenShiftTemplateConstants;
+import org.kie.cloud.openshift.scenario.KieServerWithExternalDatabaseScenarioImpl;
+
+public class KieServerWithExternalDatabaseScenarioBuilderImpl implements KieServerWithExternalDatabaseScenarioBuilder {
+
+    private OpenShiftController openshiftController;
+    private Map<String, String> envVariables;
+
+    public KieServerWithExternalDatabaseScenarioBuilderImpl(OpenShiftController openShiftController) {
+        this.openshiftController = openShiftController;
+
+        this.envVariables = new HashMap<String, String>();
+        this.envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_USER, DeploymentConstants.getKieServerUser());
+        this.envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_PWD, DeploymentConstants.getKieServerPassword());
+
+        this.envVariables.put("DB_HOST", System.getProperty("db.hostname"));
```

Comment:
Please update the variables with with OpenShiftTemplateConstants and DeploymentConstants.